omero/developers/Web/CreateApp.rst

```diff
@@ -83,7 +83,7 @@ python can import our ``minimal_webapp`` module.
     $ appdir=$(pwd)/minimal-webapp/minimal_webapp
 
     # Location within Docker instance we want to link the app, so it can be imported.
-    $ docker_appdir=/opt/omero/web/venv/lib/python2.7/site-packages/minimal_webapp
+    $ docker_appdir=/opt/omero/web/venv3/lib/python3.6/site-packages/minimal_webapp
```

Comment:
Not that it need beed done in this PR, but @manics, is there anything that could be done in the omero-web(-standalone) app to make this registration easier? A symlink from `/apps`? A `.pth` file? etc.

Reply:
Does `-e PYTHONPATH=/mountpoint` work?

Reply:
If I set $appdir so that
```
$ echo $appdir 
/Users/wmoore/Desktop/WEB-APP-EXAMPLES/omero-web-apps-examples/minimal-webapp/
```
This allows me to import the app locally:
```
$ export PYTHONPATH=$appdir
$ python
>>> import minimal_webapp
```

But this doesn't work:
```
$ docker run -it --rm -e OMEROHOST=$host -p 4080:4080 -e PYTHONPATH=$appdir -v $config:$docker_config openmicroscopy/omero-web-standalone
...
ModuleNotFoundError: No module named 'minimal_webapp'
```

Reply:
You'll still need to mount it inside the docker image. E.g.
```
appdir=$PWD/omero-metadata/src
docker run -it --rm ... -e PYTHONPATH=/apps -v $appdir:/apps --name test openmicroscopy/omero-web-standalone
```
```
$ docker exec -it test python -c 'import omero_metadata; print(omero_metadata.__file__)'
/apps/omero_metadata/__init__.pyc
```

Reply:
Created https://github.com/ome/ome-documentation/issues/2140 to look into this - but not essential for this PR.

Reply:
Thanks @will-moore, merging